buildenv/1.3.x/common/qt5/qt5.build

```diff
@@ -28,53 +29,32 @@ function prepare {
 
 	echo "# dummy" > qtbase/.gitignore
 
-	# Disable qmacpasteboardmime. There are symbol clashes with the 'cocoa' plugin,
-	# so seemingly, these two modules aren't currently engineered to be used in a
-	# static build together.
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-macextras-disable-qmacpasteboardmime.patch
-
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-force-qtimageformats-jasper-no-libjpeg.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-qtimageformats-jasper-4-color-rct-debian.patch
-
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-configureapp-use-msvc2013-mkspec.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-mariadb-support.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-mkspecs-common-msvc-desktop-mumble-debug+opt.patch
```

Comment:
I doubt it's correct to drop this outright. Elaborate?

Reply:
You're right, only the workaround for `QTBUG-58318` should be removed.